style.css

```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header h1 {
+  color: #000;
+}
```

Comment:
- [ ] Kindly make sure the `style.css` file includes one class with styles for your header [as required for the project](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=A%20.css%20file%20that%20includes%20one%20class%20with%20styles%20for%20your%20header%20(make%20it%20in%20your%20favorite%20color).). I suggest you add one class in the `<h1>` tag(in the index.html file) and then use it to style your header.